core/src/tx.rs

```diff
@@ -69,6 +65,11 @@ impl VersionedAcceptedTransaction {
         self.as_inner_v1().is_expired(transaction_time_to_live)
     }
 
+    /// If `true`, this transaction is regarded to have been tampered to have a future timestamp.
+    pub fn is_future(&self, threshold: Duration) -> bool {
```

Comment:
```suggestion
    pub fn is_future_timestamp(&self, threshold: Duration) -> bool {
```
As the word "future" has a loaded meaning in a sync rust.

Reply:
Ok, then I'll fix it to `is_in_future()` according to `Error::InFuture`